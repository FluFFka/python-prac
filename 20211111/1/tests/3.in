print(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(OrderedComplexMul(0.000000001j))))))))))))))))))))+OrderedComplex(1,2)-4j)
