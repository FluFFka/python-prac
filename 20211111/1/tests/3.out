(1-1.999999999j)
